settings.nix

```diff
@@ -34,6 +34,15 @@
 
           };
 
+          flake-lang.pre-commit-hooks.tools = {
+            rustfmt = lib.mkOption {
```

Comment:
```suggestion
            pkgsForRust = lib.mkOption {
                type = lib.types.attrs;
                readOnly = false;
                description = lib.mdDoc ''Rust packages'';
                default = import inputs.nixpkgs {
                            inherit system;
                           overlays = [ (import inputs.rust-overlay) ];
                           };
            };
            rustfmt = lib.mkOption {
```

Reply:
I think I kinda meant something like this -- kinda like avoiding modifying the `_module.args` completely. It's not that nice syntactically I suppose